app/controllers/users_controller.rb

```diff
@@ -1,3 +1,24 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = 'Welcome to the Sample App!'
+      redirect_to @user
+    else
+      render 'new'
+    end
+  end
+
+  private
+    def user_params
```

Comment:
incorrect tabulation

pls use single tabulation for `private` keyword and private functions. As here:
```
def public_function
  #some
end

private

def private_function
  #some
end
```

Need to ADD an empty line before block.